lib/patterns.coffee

```diff
@@ -83,10 +83,9 @@ exports.login = (callback) ->
 #		console.log(token)
 ###
 exports.loginWithToken = (callback) ->
-	form.ask
-		label: 'What\'s your token? (visible in the preferences page)'
-		name: 'token'
-		type: 'text'
+	form.askAuthToken
+		name: 'AuthToken'
+		message: 'What\'s your token? (visible in the preferences page)'.bold.white
```

Comment:
For consistency and encapsulation purposes, if you weren't going to inline the body of the new ask function here, `.bold.white` should have happened in the `ask` function.